docs: maintain reference quadrant (#1012)
canonical/rockcraft: docs/reference/index.rst

.. _reference:

Reference
=========

References describe the structure and function of the individual components in
Rockcraft.

Commands
--------

Rockcraft is operated from the command line, with a command for each function.

:ref:`reference-commands`

Project file
------------

The main object inside an Rockcraft project is a configurable project file. Read on
for a complete reference of this file's structure and contents.

:ref:`reference-rockcraft-yaml`

Parts
-----

> Software is brought into an image through the declaration of parts. Each part must be

Reviewer: ```suggestion
Software is brought into a rock through the declaration of parts. Each part must be
```
